boards/omnigul/omnigul.config

```diff
@@ -0,0 +1,52 @@
+# Configuration for a Acer Chromebook Plus 515 (CB515-2H/OMNIGUL)
```

Comment:
Document HCL of latop including graphic card and specs for your owned model.

Reply:
I attached it. Let me know what else is needed.